acceptance/helpers/beaker_helper.rb

```diff
@@ -280,6 +349,37 @@ def hosts_without_role(hosts, role_to_exclude)
       host unless host["roles"].include?(role_to_exclude.to_s)
     end
   end
+
+  # Runs the command on the specified host
+  #
+  # @param host The unix style host
+  # @param command The command to run
+  #
+  # @example
+  #   output = run_command(host, command)
+  #
+  def run_beaker_command(host, command)
+    puts "command: #{command}"
+    output = on(host, command).stdout.rstrip
+    puts "output:"
+    puts output
+    return output
+  end
+
+  # Runs the mock puppet agent on the specified host
+  #
+  # @param host The unix style host
+  #
+  # @example
+  #   output = run_mock_puppet_agent(host)
+  #
+  def run_mock_puppet_agent(host)
```

Comment:
I thought the fake install script created a link or something so that running puppet agent -t would run the thing in the tarball without us having to do that... If not... that might be a good tech debt ticket, so that we don't have to have this special mock out.

Reply:
Updated puppet-enterprise-installer in the fake tarball to copy puppet from '/tmp/ref_arch_setup/puppet-enterprise-*/puppet'